### PR TITLE
SW status information

### PR DESCRIPTION
**Pull request by denar90**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -97,10 +98,27 @@ <h3 class="f4 br3 br--top black-60 pv2 mb0">Wanna compare traces?</h3>
 
         // We are monkeypatching window.loadResourcePromise, which is from devtools' Runtime.js
         viewer.devTools.monkeypatchLoadResourcePromise();
+
+        if ('serviceWorker' in navigator) {
```
Comment (paulirish) [inline docs/index.html]: did this come by way of CRA?

saw another app that had nearly identical code..

Comment (denar90) [inline docs/index.html]: Yep it's kinda copy pasta from https://github.com/facebookincubator/create-react-app/blob/b98ba3d7ea21a7c0cb4e250559dbe0e44b89698c/packages/react-scripts/template/src/registerServiceWorker.js#L15-L41
But we can copy paste it from https://github.com/GoogleChrome/sw-precache/blob/7b1b4f84c69fb7ae88d03b66e6a97be7955f0f79/demo/app/js/service-worker-registration.js#L29-L60 😄

File: docs/index.html [hunk 2]
```diff
@@ -97,10 +98,27 @@ <h3 class="f4 br3 br--top black-60 pv2 mb0">Wanna compare traces?</h3>
...
+          navigator.serviceWorker
+            .register('service-worker.js')
+            .then(registration => {
+              registration.onupdatefound = () => {
+                const installingWorker = registration.installing;
+                installingWorker.onstatechange = () => {
+                  if (installingWorker.state === 'activated') {
```
Comment (denar90) [inline docs/index.html]: btw, are you ok if the message will be shown when SW was activated? Looks like user will see it only once.

Comment (paulirish) [inline docs/index.html]: seems good. :)

nice

**Comment by paulirish** [comment]
got a conflict now. do we still have an init() ?

im happy to resolve this.

**Comment by paulirish** [comment]
merged!

**Pull request merged**